src/lib/OTA/OTA.h

```diff
@@ -17,20 +21,73 @@
 // Mask used to XOR the ModelId into the SYNC packet for ModelMatch
 #define MODELMATCH_MASK 0x3f
 
+#define SYNC_PACKET_SWITCH_OFFSET   1   // Switch encoding mode
+#define SYNC_PACKET_TLM_OFFSET      3   // Telemetry ratio
+#define SYNC_PACKET_RATE_OFFSET     6   // Rate index
+#define SYNC_PACKET_SWITCH_MASK     0b11
+#define SYNC_PACKET_TLM_MASK        0b111
+#define SYNC_PACKET_RATE_MASK       0b11
+
+typedef struct {
+    uint8_t type: 2,
+            crcHigh: 6;
+    union {
+        struct {
+            uint8_t fhssIndex;
+            uint8_t nonce;
+            uint8_t free:1,
+                    switchEncMode:2,
+                    newTlmRatio:3,
+                    rateIndex:2;
+            uint8_t UID3;
+            uint8_t UID4;
+            uint8_t UID5;
+        } sync;
+        struct {
+            uint8_t ch0High;
+            uint8_t ch1High;
+            uint8_t ch2High;
+            uint8_t ch3High;
+            uint8_t ch3Low:2,
+                    ch2Low:2,
+                    ch1Low:2,
+                    ch0Low:2;
+            uint8_t switches;
+        } rc;
+        struct {
+            uint8_t packageIndex;
+            uint8_t data[5];
+        } msp;
+        struct {
+            uint8_t type;
+            uint8_t uplink_RSSI_1:7,
+                    antenna:1;
+            uint8_t uplink_RSSI_2:7,
+                    modelMatch:1;
+            uint8_t SNR;
+            uint8_t lq;
+            uint8_t mspConfirm:1,
+                    free:7;
+        } tlm;
```

Comment:
My suggestion for tlm would be link_statistics

Reply:
thanks! that makes more sense.